src/status_im/ui/screens/wallet_connect/session_proposal/views.cljs

```diff
@@ -0,0 +1,215 @@
+         (vector dapps-account)
+         selected-account
+         {:on-press #(do
+                       (re-frame/dispatch [:wallet-connect/manage-app session])
+                       (reset! show-account-selector? true))}]
+        [quo/text {:weight :regular
+                   :color :secondary
+                   :style  styles/message-title}
+         (i18n/label :t/manage-connections)]
+        [react/view (styles/footer)
+         [react/view styles/success-button-container
+          [react/view styles/proposal-button-right
+           [quo/button
+            {:theme     :accent
+             :on-press  #(do
+                           (reset! show-account-selector? false)
+                           (re-frame/dispatch [:hide-wallet-connect-success-sheet]))}
+            (i18n/label :t/close)]]]]]
+       (when managed-session
+         (if platform/ios?
+           [react/blur-view {:style (styles/blur-view)
+                             :blurAmount 2
+                             :blurType (if (colors/dark?) :dark :light)}]
+           [react/view (styles/blur-view)]))])))
+
+(defview app-management-sheet-view [{:keys [topic]}]
+  (letsubs [sessions [:wallet-connect/sessions]
+            visible-accounts [:visible-accounts-without-watch-only]]
+    (let [{:keys [peer state]} (first (filter #(= (:topic %) topic) sessions))
+          {:keys [accounts]} state
+          {:keys [metadata]} peer
+          {:keys [name icons url]} metadata
+          icon-uri (when (and icons (> (count icons) 0)) (first icons))
+          account-address (last (string/split (first accounts) #":"))
+          selected-account (reagent/atom (first (filter #(= (:address %) account-address) visible-accounts)))]
+      [react/view {:style (merge (styles/acc-sheet) {:background-color "rgba(0,0,0,0)"})}
+       [react/linear-gradient {:colors ["rgba(0,0,0,0)" "rgba(0,0,0,0.3)"]
+                               :start {:x 0 :y 0} :end {:x 0 :y 1}
+                               :style styles/shadow}]
+       [react/view (styles/proposal-sheet-container)
+        [react/view (styles/management-sheet-header)
+         [react/image {:style styles/management-icon
+                       :source {:uri icon-uri}}]
+         [react/view styles/app-info-container
+          [quo/text {:weight :medium} name]
+          [quo/text {:color :secondary
+                     :number-of-lines 1
+                     :elipsize-mode :tail} url]]
+         [quo/button
+          {:type :secondary
+           :theme :secondary
+           :on-press #(re-frame/dispatch [:wallet-connect/disconnect topic])}
+          (i18n/label :t/disconnect)]]
+        [account-selector
+         visible-accounts
+         selected-account
+         #(re-frame/dispatch [:wallet-connect/change-session-account topic @selected-account])]]])))
+
+(defview session-proposal-sheet [{:keys [name icons]}]
+  (let [visible-accounts @(re-frame/subscribe [:visible-accounts-without-watch-only])
```

Comment:
if you use defview, it makes sense to use letsubs and remove `@(re-frame/subscribe`

Reply:
Absolutely. Fixed